language/vm/vm-runtime/vm-runtime-types/src/values/values_impl.rs

```diff
@@ -0,0 +1,2223 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    loaded_data::{struct_def::StructDef, types::Type},
+    native_functions::dispatch::{native_gas, NativeResult},
+    native_structs::def::{NativeStructTag, NativeStructType},
+};
+use itertools::Itertools;
```

Comment:
can we please remove this?
We really want to avoid adding random dependencies unless strictly needed and I don't believe we need this